### DISCUSSION TITLE
Trusted proxies does not work on nextcloud-aio-nextcloud for reverse proxy nginx docker

### DISCUSSION THREAD

**Discussion by javaloyesgithub** [nextcloud/all-in-one#4289] [Questions] [answered]
Hello everyone,
I have a fresh all in one nextcloud installation on a Raspberry pi 5 with 64bit OS and one reverse proxy on nginx docker following documentation on this post https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md. I was trying to add my reverse proxy docker ip following this post https://github.com/nextcloud/all-in-one/discussions/2045 with command

**Comment by szaimen** [comment, marked as answer]
Hi, which Ip-address is logged in the nextcloud logs available via `https://yourdomain.com/settings/admin/logging` when a login has failed?

**Comment by javaloyesgithub** [comment]
Hello @szaimen , first all, thank you very much for your answer and support. 
When I try to access via web to https://nextcloud.javaloyesengineering.com/settings/admin/logging I get 502 error.
And I get this error log from my docker nginx-front reverse-proxy:

**Reply by szaimen** [reply]
Can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?
